drivers/storage/portworx/component/plugin.go

```diff
@@ -354,3 +360,23 @@ func isVersionSupported(v string) bool {
 
 	return currentVersion.GreaterThanOrEqual(targetVersion)
 }
+
+func getDesiredPluginImage(cluster *corev1.StorageCluster) string {
+	imageName := "portworx/portworx-dynamic-plugin:1.0.0"
+
+	if cluster.Status.DesiredImages != nil && cluster.Status.DesiredImages.DynamicPlugin != "" {
+		imageName = cluster.Status.DesiredImages.DynamicPlugin
+	}
+	imageName = util.GetImageURN(cluster, imageName)
+	return imageName
+}
+
+func getDesiredNginxcImage(cluster *corev1.StorageCluster) string {
+	imageName := "nginxinc/nginx-unprivileged:1.23"
```

Comment:
Same as before -- let's declare it as `const` and (if possible) move to `manifest.go`

Reply:
done